docs/table/filter.rst

```diff
@@ -11,80 +11,146 @@ extract portions of a table based on some criteria, this is called filtering.
+   >>> end = to_gps("Jul 2019")
+   >>> def q12_2019(column, interval):
+   ...     """Returns `True` if ``interval[0] <= column < interval[1]``
+   ...     """
+   ...     return (column >= interval[0]) & (column < interval[1])
+   >>> print(events.filter(('GPS', q12_2019, (start, end))))
+          name        chi_eff_upper ...     GPS      final_mass_source_upper
+                                    ...                      solMass
+   ------------------ ------------- ... ------------ -----------------------
+   GW190408_181802-v1          0.14 ... 1238782700.3                     3.9
+          GW190412-v3          0.08 ... 1239082262.2                     3.9
+   GW190413_052954-v1          0.29 ... 1239168612.5                    12.5
+                  ...           ... ...          ...                     ...
+   GW190706_222641-v1          0.26 ... 1246487219.3                    18.3
+   GW190707_093326-v1           0.1 ... 1246527224.2                     1.9
+   GW190708_232457-v1           0.1 ... 1246663515.4                     2.5
+   Length = 24 rows
 
 The custom filter function could have been as complicated as we liked, as long
 as the two (and only two) input arguments were the column array for the
 relevant column, and the collection of other arguments to work with.
-
-Similarly, we could filter the catalogue to find only those events that include
-data from the Virgo observatory::
-
-    >>> import numpy
-    >>> print(table.filter(('network', numpy.char.endswith, 'V')))
-      name        gps       m1   m2  snr  network
-    -------- ------------- ---- ---- ---- -------
-    GW170814 1186741861.53 30.5 25.3 18.0     HLV
+For example could filter the table to return only those events with
+high mass ratio:
+
+.. code-block:: python
+   :name: gwpy-table-filter-function-example-2
+   :caption: Filtering an `EventTable` using multiple columns
+
+   >>> def high_mass_ratio(table, threshold):
+   ...     """Returns `True` if ``mass_1_source / mass_2_source >= threshold``
+   ...     """
+   ...     return (table['mass_1_source'] / table['mass_2_source']) >= threshold
+   >>> print(events.filter((('mass_1_source', 'mass_2_source'), high_mass_ratio, 3.0)))
+          name        chi_eff_upper ...     GPS      final_mass_source_upper
+                                    ...                      solMass
+   ------------------ ------------- ... ------------ -----------------------
+          GW190412-v3          0.08 ... 1239082262.2                     3.9
+   GW190426_152155-v1          0.32 ... 1240327333.3                    None
+          GW190814-v2          0.06 ... 1249852257.0                     1.1
+   GW190929_012149-v1          0.34 ... 1253755327.5                    33.6
 
 ======================
 Using multiple filters
 ======================
 
-Filters can be trivially chained (either in `str` form, or functional form)::
+Filters can be trivially chained (either in `str` form, or functional form):
```

Comment:
Minor suggestion: remove the word "trivially" as the word doesn't really aid the documentation, and can feel demoralizing to someone who might struggle to make use of the multiple filtering

Reply:
Agree 100%, will change.